cypress/integration/examples/cypress_api.spec.js

```diff
@@ -170,7 +176,10 @@ context('Cypress.env()', () => {
 
     // get all environment variable
     expect(Cypress.env()).to.have.property('host', 'veronica.dev.local')
-    expect(Cypress.env()).to.have.property('api_server', 'http://localhost:8888/v2/')
+    expect(Cypress.env()).to.have.property(
+      'api_server',
+      'http://localhost:8888/v2/'
```

Comment:
Avoid hardcoded environments, move to variable or config environment file

Reply:
file deleted since this is sample test only